src/runtime.js

```diff
@@ -513,6 +513,36 @@ export class MultidocManager {
       amp.resources = Services.resourcesForDoc(ampdoc);
     }
 
+    /**
+     * Expose amp-bind methods
+     */
+    amp['bind'] = {
+      /**
+       * Expose amp-bind getState
+       * @param {string} name - Name of state or deep state
+       * @return {Promise<(?JsonObject|string|undefined)>} - Resolves to
+       * a copy of the value of a state
+       */
+      getState: name => {
+        return Services.bindForDocOrNull(shadowRoot).then(bind => {
```

Comment:
For pages that do not use `amp-bind`, this promise never resolves. I don't think it's a problem, but wanted to make sure this received consideration.

Reply:
Hm, it should resolve with `null` on pages that are missing the extension script in the doc head.